### PR TITLE
New infrastructure

### PR DESCRIPTION
**Pull request by rstens**
*Issue:*https://github.com/bcgov/entity/issues/22356

*Description of changes:*
E2E Testing infrastructure

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-89-pfxlaffp.web.app

**Comment by rstens** [comment]
Needs /gcbrun

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://strr-ui-dev--pr-89-pfxlaffp.web.app

File: testing/README.md
Comment (dimak1) [inline testing/README.md:1]: Awesome, thanks for detailed instructions 👍🏻

**Review by dimak1 — APPROVED** [review]
Great work!

**Pull request merged**